### ISSUE TITLE
FashionCLIP meta tensor loading error in ai-enrichment

### ISSUE THREAD

**Issue by mikesvoboda** [mikesvoboda/nemotron-v3-home-security-intelligence#460]
## Issue

FashionCLIP (clothing classifier) fails to load in ai-enrichment container with error:
`Cannot copy out of meta tensor; no data! Please use torch.nn.Module.to_empty() instead of torch.nn.Module.to() when moving module from meta to a different device.`

## Root Cause

The open_clip library's model loading mechanism uses meta tensors which require special handling when moving to GPU.

## Current State

* ai-enrichment service starts but FashionCLIP is not loaded
* Vehicle classifier and Pet classifier work fine
* Health endpoint shows 'degraded' status

## Fix Options

1. Use open_clip's proper loading API with device placement
2. Pre-download model weights and load directly
3. Use transformers AutoModel instead of open_clip

## Logs

```
2026-01-01 21:47:26,616 - __main__ - ERROR - Failed to load clothing classifier: Cannot copy out of meta tensor; no data!
```

## Labels

model-zoo, ai-enrichment, bug

---

*Migrated from beads:* `home_security_intelligence-d9qk`
*Original created: 2026-01-01*
*Original closed: 2026-01-01*
